style.css

```diff
@@ -27,7 +27,7 @@
     justify-content: space-between;
   }
 
-  .desktop-add-heading::before {
+  /* .desktop-add-heading::before {
```

Comment:
Please remove these lines of commented code to keep a cleaner codebase.

Reply:
removed.